app.py

```diff
@@ -1,8 +1,20 @@
-from flask import Flask, render_template
+from flask import Flask, render_template, json, redirect, request
+from flask_mysqldb import MySQL
+import os
 
 
 app = Flask(__name__)
 
+# database connection
+app.config["MYSQL_HOST"] = "localhost"

```

Comment:
MYSQL_HOST, USER, PASSWORD, and DB will all need to be changed to the OSU hosted credentials (And I did have a user named test with password test that I was using locally, shame on me 😞)